include/hpp/constraints/static-stability.hh

```diff
@@ -50,16 +50,13 @@ namespace hpp {
         static const Eigen::Matrix <value_type, 6, 1> Gravity;
 
         struct Contact_t {
-          JointPtr_t joint1, joint2;
-          vector3_t point1, point2;
-          vector3_t normal1, normal2;
```

Comment:
I think I did consider a contact between two moving joints at some point. I guess you change this because I didn't implement it, did I ?

Reply:
No only `joint2`, `point2` and `normal2` were used.

Reply:
Ok, I remember now. Actually, a contact between two moving joints should be handled differently.
- (implemented) make sure the two joints are in contact and share the same normal
- (not implemented) in this class, consider each moving joint as an independent contact and constrain the magnitude of each force to be equal (the normal being opposite).